data-reporting-service/build.sh

```diff
@@ -25,15 +25,21 @@ echo "Building Kafka Broker"
 docker-compose -f $BASE/docker-compose.yml up broker --build -d
 
 # Build and deploy ETL Data pipeline container
-echo "Building ETL Data Pipeline"
+echo "Updating the th Data Pipeline"
```

Comment:
Can we keep the comment as "Building the-"

Reply:
Good catch!